package.json

```diff
@@ -79,6 +79,8 @@
     "@types/sizzle": "^2.3.2",
     "@types/webpack": "^4.41.26",
     "@types/ws": "^7.4.0",
+    "@types/yarnpkg__lockfile": "^1.1.5",
+    "@yarnpkg/lockfile": "^1.1.0",
```

Comment:
Can you add `@yarnpkg/lockfile` to https://github.com/ionic-team/stencil/blob/66d0476bb8acff5225926cc50769db16a5fd8ed3/scripts/license.ts#L12

Reply:
Truthfully? /shrug

Reply:
It's a part of the compiler process, since anything within the node sys gets bundled up and shipped. Similar to how we bundle TypeScript or Rollup.